forms-flow-idm/keycloak/docker-compose.yml

```diff
@@ -25,15 +25,14 @@ services:
       - keycloak-server-network
 
   keycloak:
-    image: jboss/keycloak:14.0.0
+    image: formsflow/keycloak:latest
```

Comment:
Please do not use keycloak latest; as the spec has changed in v15 and breaks BPM

Reply:
And btw; do we have keycloak image under formsflow org ?

Reply:
And it's best to not build our own custom images if it's not really needed